npm/install.js

```diff
@@ -15,6 +15,10 @@ try {
   // do nothing
 }
 
+if (process.env['ELECTRON_BINARY_SKIP_DOWNLOAD']) {
```

Comment:
This doesn't have to be quoted

```suggestion
if (process.env.ELECTRON_BINARY_SKIP_DOWNLOAD) {
```